HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,96 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly byte[] _keyBytes = new byte[32];
+    private readonly string _issuer;
+    private readonly string _audience;
```

Comment:
When you migrate the config here to use `IOptions` only store the `IOptions` here.